plutus-playground/plutus-playground-client/src/Playground/API.purs

```diff
@@ -0,0 +1,170 @@
+-- File auto generated by purescript-bridge! --
```

Comment:
I don't suppose it's convenient to do the generation as part of the build rather than checking it in?

Reply:
it is if you do nix. this isn't supposed to be checked in, I've fixed the gitignore